### PR TITLE
MariaDB mentions

### PR DESCRIPTION
**Pull request by ottok**
## Describe this PR

I have never used Prisma before, and it took me quite a lot of extra effort to figure out if Prisma supports MariaDB at all because the documentation was only mentioning MySQL. After some digging, I found that indeed Prisma uses both https://www.npmjs.com/package/mariadb for the connector and https://hub.docker.com/_/mariadb as part of the CI, and MariaDB support is actually excellent.

## What issue does this fix?

The docs hide the fact that Prisma actually supports MariaDB. [Sequalize has very clear MariaDB documentation](https://sequelize.org/docs/v7/other-topics/dialect-specific-things/#mariadb), and if you merge this PR then Prisma will be much more clear about the MariaDB support situation.

## Changes

Each commit is [atomic](https://optimizedbyotto.com/post/good-git-commit/#1-commits-should-be-atomic) and carefully documented - please review this PR by looking commit-by-commit looking at the message and the change at the same time. 

Looking at [git log in 500-databases](https://github.com/prisma/docs/commits/main/content/200-orm/050-overview/500-databases) and e.g. 4358a4169ef3ff042d70b825a9cf3e16bf71e48e seems you prefer to squash when merging. Please merge this PR as-is **without** squashing to avoid loosing the per-commit documentation.

## Any other relevant information

I am also happy to iterate on the contents of this PR based on your feedback. I am also happy to rebase frequently.

### CODE REVIEW

**Comment by jharrell** [comment]
Hi there @ottok , is this PR distinct from #5706 ?

**Comment by ottok** [comment]
Yes, I split this and also #5716 out from #5706 in hope that it is easier for you to review and to accept a few smaller PRs instead of one big one.

**Comment by jharrell** [comment]
Hey there @ottok thanks again for your pull requests. 

We want to pull some of this in, but not all of it. Would you mind changing this PR to only have the changes to these files?

- content/200-orm/100-prisma-schema/20-data-model/20-relations/410-referential-actions/index.mdx
- content/200-orm/500-reference/375-supported-databases.mdx
- src/components/shortcodes/index.tsx

These are references where we think that, indeed, we also need to mention MariaDB. We can then review that and get it merged in. 

As for #5706 , we can't accept it in its current state. MariaDB is supported via the MySQL provider, so having duplicated documentation that can quickly get out of sync isn't something we want to do. 

Instead, we'd like to change the title of `content/200-orm/050-overview/500-databases/400-mysql.mdx` to `MySQL and MariaDB` from just `MySQL`. An additional sentence describing that we also support MariaDB due to it being compatible with MySQL would be nice as well. 

I hope that all makes sense! Thank you again for your work.

**Comment by ottok** [comment]
Thanks for the feedback! I have updated the PR following your guidance.

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/index.mdx [hunk 1]
```diff
@@ -39,7 +39,7 @@ In order to successfully complete this guide, you need:
 
 - an existing Node.js project with a `package.json`
 - [Node.js](https://nodejs.org/en/) installed on your machine
-- a [MySQL](https://www.mysql.com/) database server running and a database with at least one table
+- a [MySQL](https://www.mysql.com/) or [MariaDB](https://mariadb.org) database server running and a database with at least one table
```
Comment (jharrell) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/index.mdx]:
```suggestion
- a [MySQL](https://www.mysql.com/) database server running and a database with at least one table
```

File: content/200-orm/050-overview/500-databases/400-mysql.mdx [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
-title: 'MySQL'
+title: 'MySQL/MariaDB'
 metaTitle: 'MySQL database connector'
-metaDescription: 'This page explains how Prisma can connect to a MySQL database using the MySQL database connector.'
+metaDescription: 'This page explains how Prisma can connect to a MySQL/MariaDB database using the MySQL database connector.'
```
Comment (jharrell) [inline content/200-orm/050-overview/500-databases/400-mysql.mdx]:
```suggestion
metaDescription: 'This page explains how Prisma can connect to a MySQL or MariaDB database using the MySQL database connector.'
```

File: src/components/techSwitcher.tsx [hunk 3]
```diff
@@ -52,7 +52,7 @@ const technologyTypes = {
 export const technologyNames = {
   node: 'JavaScript',
   typescript: 'TypeScript',
-  mysql: 'MySQL',
+  mysql: 'MySQL/MariaDB',
```
Comment (jharrell) [inline src/components/techSwitcher.tsx]:
```suggestion
  mysql: 'MySQL',
```

**Review by jharrell — CHANGES_REQUESTED** [review]

**Comment by ottok** [comment]
Thanks Jon for the feedback - I have updated the PR accordingly and also rebased on latest prisma:main.

Looking at [git log in 500-databases](https://github.com/prisma/docs/commits/main/content/200-orm/050-overview/500-databases) and e.g. https://github.com/prisma/docs/commit/4358a4169ef3ff042d70b825a9cf3e16bf71e48e seems you prefer to squash when merging. Please merge this PR as-is without squashing to avoid loosing the per-commit documentation. Each commit is [atomic](https://optimizedbyotto.com/post/good-git-commit/#1-commits-should-be-atomic) and carefully documented.

**Comment by jharrell** [comment]
> Looking at [git log in 500-databases](https://github.com/prisma/docs/commits/main/content/200-orm/050-overview/500-databases) and e.g. [4358a41](https://github.com/prisma/docs/commit/4358a4169ef3ff042d70b825a9cf3e16bf71e48e) seems you prefer to squash when merging. Please merge this PR as-is without squashing to avoid loosing the per-commit documentation. Each commit is [atomic](https://optimizedbyotto.com/post/good-git-commit/#1-commits-should-be-atomic) and carefully documented.

@ottok is there any particular reason why this needs to be _not_ squashed and merged? I appreciate the work you did here and am happy to merge, but we squash/merge as a default. Commits are still available in the respective PR.

**Comment by ottok** [comment]
> > Looking at [git log in 500-databases](https://github.com/prisma/docs/commits/main/content/200-orm/050-overview/500-databases) and e.g. [4358a41](https://github.com/prisma/docs/commit/4358a4169ef3ff042d70b825a9cf3e16bf71e48e) seems you prefer to squash when merging. Please merge this PR as-is without squashing to avoid loosing the per-commit documentation. Each commit is [atomic](https://optimizedbyotto.com/post/good-git-commit/#1-commits-should-be-atomic) and carefully documented.
> 
> @ottok is there any particular reason why this needs to be _not_ squashed and merged? I appreciate the work you did here and am happy to merge, but we squash/merge as a default. Commits are still available in the respective PR.

If you care about linear history, or about being able to read git commits and the messages through tools like `git blame` etc, you should prefer linear history and normal merges or [rebase+merge](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/about-merge-methods-on-github#rebasing-and-merging-your-commits). But this is just a sidetrack for the doc change - git use is a topic for another day. Feel free to merge as you wish. 

Thanks for the feedback rounds to get this polished, the result will now make the docs much better for MariaDB users. Thanks!

**Review by jharrell — APPROVED** [review]

**Pull request merged**